build_package.py

```diff
@@ -37,24 +52,46 @@
 base_wheel_location = glob.glob("dist/*.whl")[0]
 without_platform = base_wheel_location[:-7]
 
-pack_wheel_drivers = []
-if sys.platform == "linux":
-    pack_wheel_drivers.append(("driver-linux", "manylinux1_x86_64.whl"))
-if sys.platform == "darwin":
-    pack_wheel_drivers.append(("driver-darwin", "macosx_10_13_x86_64.whl"))
-if sys.platform == "win32":
-    # Windows is the only one that can build all drivers (x86 and x64),
-    # so we publish from it
-    pack_wheel_drivers.append(("driver-linux", "manylinux1_x86_64.whl"))
-    pack_wheel_drivers.append(("driver-darwin", "macosx_10_13_x86_64.whl"))
-    pack_wheel_drivers.append(("driver-win32.exe", "win32.whl"))
-    pack_wheel_drivers.append(("driver-win32-amd64.exe", "win_amd64.whl"))
-
-for driver, wheel in pack_wheel_drivers:
+platform_map = {
+    "darwin": "mac",
+    "linux": "linux",
+    "win32": "win32_x64" if sys.maxsize > 2 ** 32 else "win32",
+}
+
+for platform in ["mac", "linux", "win32", "win32_x64"]:
+    zip_file = f"driver/playwright-cli-{driver_version}-{platform}.zip"
+    with zipfile.ZipFile(zip_file, "r") as zip:
+        zip.extractall(f"driver/{platform}")
+    if platform_map[sys.platform] == platform:
+        with zipfile.ZipFile(zip_file, "r") as zip:
+            zip.extractall("playwright/driver")
+        for file in os.listdir("playwright/driver"):
+            if file == "playwright-cli" or file.startswith("ffmpeg"):
+                print(f"playwright/driver/{file}")
+                os.chmod(
+                    f"playwright/driver/{file}",
+                    os.stat(f"playwright/driver/{file}").st_mode | stat.S_IEXEC,
+                )
+
+    wheel = ""
+    if platform == "mac":
+        wheel = "macosx_10_13_x86_64.whl"
+    if platform == "linux":
+        wheel = "manylinux1_x86_64.whl"
+    if platform == "win32":
+        wheel = "win32.whl"
+    if platform == "win32_x64":
+        wheel = "win_amd64.whl"
     wheel_location = without_platform + wheel
     shutil.copy(base_wheel_location, wheel_location)
-    from_location = f"driver/out/{driver}"
-    to_location = f"playwright/drivers/{driver}"
-    with zipfile.ZipFile(wheel_location, "a") as zipf:
-        zipf.write(from_location, to_location)
+    with zipfile.ZipFile(wheel_location, "a") as zip:
+        for file in os.listdir(f"driver/{platform}"):
```

Comment:
Got it. So driver will get still distributed over wheel. Then curl is fine, maybe we should download it on-demand which seems better (source code install would work).

Reply:
Yes, we still bundle.